components/dashboard/src/teams/sso/OIDCClientListItem.tsx

```diff
@@ -111,6 +166,33 @@ export const OIDCClientListItem: FC<Props> = ({ clientConfig }) => {
             {showEditModal && (
```

Comment:
**issue(non-blocking)**: Editing an active configuration leads to a blocked step, as you can't do much with the configuration in the modal. Do we still need the edit there?

| More actions on an active configuration | Edit modal for an active configuration |
|-|-|
| <img width="1512" alt="Screenshot 2023-05-16 at 23 20 51" src="https://github.com/gitpod-io/gitpod/assets/120486/66f4736c-61c0-4c20-a599-35f2ee389d2c"> | <img width="1512" alt="Screenshot 2023-05-16 at 23 20 53" src="https://github.com/gitpod-io/gitpod/assets/120486/8825d189-acc7-47cc-b065-bc68bcc1714a"> |

Reply:
Maybe the term Edit is misleading here now. We could change it in the menu to "View". Thoughts? I think there's some value in being able to pop it back open, and reference the configuration and redirect url.